src/core/qgsproject.h

```diff
@@ -500,9 +500,32 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
      */
     QStringList readListEntry( const QString &scope, const QString &key, const QStringList &def = QStringList(), bool *ok = nullptr ) const;
 
+    /**
+     * \returns entry value as string from \a scope given its \a key
+     * \param def returned value if key doesn't exist
+     * \param ok set to TRUE if key exists and has been successfully retrieved
+     */
     QString readEntry( const QString &scope, const QString &key, const QString &def = QString(), bool *ok = nullptr ) const;
```

Comment:
Can you mark ok as "SIP_OUT" in here so that the python docs are correctly generated?

Reply:
Isn't that an API change?

Reply:
Doesn't seem to be - check the test results from this pr, sip is already treating ok as a return argument. It's just sipify which needs the hints for the documentation.

Reply:
indeed, there is no API change. I added SIP_OUT